test/js/bun/test/only-fixture-1.ts

```diff
@@ -0,0 +1,11 @@
+test.only("only this one should run", () => {
+  expect().pass();
```

Comment:
This doesn't actually verify the code ran, it would be better to `console.log` and check the output.

Reply:
We test the number of tests ran by the output